ocean_lib/web3_internal/contract_handler.py

```diff
@@ -28,11 +29,25 @@ class ContractHandler(object):
 
     @staticmethod
     def get_contracts_addresses(network, address_file):
-        if not address_file or not os.path.exists(address_file):
+        # fill addresses - across all networks
+        if not address_file:
             return None
-        with open(address_file) as f:
-            addresses = json.load(f)
+        elif str(address_file)[:8] == "https://":
```

Comment:
maybe use `elif address_file.startswith("https://"):`
also don't need the `elif` because of the previous return, `if` will do here